Modix.Services/Starboard/StarboardService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using Discord;
+using Discord.WebSocket;
+using Modix.Services.Core;
+using Modix.Data.Models.Core;
+
+namespace Modix.Services.Starboard
+{
+    public class StarboardService
```

Comment:
Might be nice to have a corresponding interface like the other services have, and some documentation comments for the public methods.